gems/pending/appliance_console.rb

```diff
@@ -289,11 +290,14 @@ module ApplianceConsole
 
           if agree("Apply time and timezone configuration? (Y/N): ")
             say("Applying time and timezone configuration...")
-            Env['TIMEZONE'] = new_loc, new_city
-            Env['TIME'] = new_date, new_time unless Env.error?
+            res = AwesomeSpawn.run("timedatectl set-timezone \"#{new_loc}/#{new_city}\"")
+            if res.exit_status == 0
+              res = AwesomeSpawn.run("timedatectl set-time \"#{new_date} #{new_time}\"",
+                                     :params => {"adjust-system-clock".to_sym => nil})
+            end
```

Comment:
doing all the manipulation of strings goes against what AwesomeSpawn provides...same goes for the exit_status...how about this?

```ruby
begin
  AwesomeSpawn.run!("timedatectl", :params => ["set-timezone", "#{new_loc}/#{new_city}"])
  AwesomeSpawn.run!("timedatectl", :params => ["set-time", "#{new_date} #{new_time}", :adjust_system_clock])
rescue AwesomeSpawn::CommandResultError
  say("Failed to apply time and timezone configuration")
end
```